src/applications/check-in/utils/appointment/index.js

```diff
@@ -165,6 +165,60 @@ const sortAppointmentsByStartTime = appointments => {
     : [];
 };
 
+/**
+ * @param {Array<Appointment>} appointments
+ */
+
+function organizeAppointmentsByYearMonthDay(appointments) {
+  const organizedData = [];
+  const months = [
```

Comment:
These months need to be translatable

Reply:
I refactored this so it doesn't rely on the keys for display